### PR TITLE
Merging the initial High Level requirements

### PR DESCRIPTION
**Pull request by acorbin3**
Adding the initial high level requirements in a separate branch and creating a pull request to be merged into the Math_library branch.

### CODE REVIEW

File: High_Level_Requirements [hunk 1]
```diff
@@ -0,0 +1,5 @@
+High Level requirements for the math library
+Math features
+Sine - This featute shall take an input value and calculate and return the sine value
+Cosine - This featute shall take an input value and calculate and return the sine value
+Floor - TBD
```
Comment (acorbin3) [inline High_Level_Requirements]: This needs to be filled out

**Review by acorbin3 — COMMENTED** [review]
Agree to the changes

**Comment by acorbin3** [comment]
This has been completed but we need someone to review it

**Comment by acorbin3** [comment]
This PR is associated to #1

File: High_Level_Requirements [hunk 2]
```diff
@@ -0,0 +1,5 @@
+High Level requirements for the math library
+Math features
+Sine - This featute shall take an input value and calculate and return the sine value
```
Comment (jlsilvey) [inline High_Level_Requirements]: Feature is mis-spelled. Also we should add some context that allows people to see if there are LLRs associated with it.  Right now, i would say that there needs to be some range bounds defined somewhere (usually in an LLR), but I dont know from looking at this screen if there are LLRs defined.

File: High_Level_Requirements [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Cosine - This featute shall take an input value and calculate and return the sine value
```
Comment (jlsilvey) [inline High_Level_Requirements]: Feature mis-spelled, also should return cosine value, not sine value.

Comment (acorbin3) [reply High_Level_Requirements]: - [x] Will update spelling

Comment (acorbin3) [reply High_Level_Requirements]: - [x] And update requirement to remove the correct value

File: High_Level_Requirements (hunk 2 above)
Comment (acorbin3) [reply High_Level_Requirements]: - [x] Will update spelling.

From this screen we will only see changes between branches. You could always look a the file by pressing the button "View file" 
In the LLRs I did state that this function shall take in degree's and radians but that was un-bounded.  Should bounds be defined at the HLR level?

Comment (jlsilvey) [reply High_Level_Requirements]: bounds would normally be defined in the LLRs.... its hard to see the full trace from here, and i didnt see the trace in the view file view.... i'm sure you showed me this before but i'm not seeing it now....

Comment (acorbin3) [reply High_Level_Requirements]: image
Here is where to view the file. This is could be helpful if you wanted to see the whole file and if this only showed the Diffs

- [x] I will add comments for the links. 

This isnt exactly how doxygen requirements would be represented. Ideally we would have a links related to this requirement where there was a reference to the LLR's. You would have to manually trace it within the repo or I would envision people would go to the rendered Doxygen to be able to do the tracing.

Comment (acorbin3) [reply High_Level_Requirements]: Added links but I also had to add unique id's in the LLRs. That change was based on this commit 520fa773f399f4a5ab8eff4c11d170d5c1000a88

Comment (acorbin3) [reply High_Level_Requirements]:
```suggestion
Sine - This feature shall take an input value and calculate and return the sine value
```

**Pull request merged**